### PR TITLE
Update delegate.md

### PR DESCRIPTION
**Pull request by thisrohangupta**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/delegate.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ import Helmdep from '/release-notes/shared/helm-2-deprecation-notice.md'
 - The List Tokens API now supports listing all the personal access tokens or service account tokens in the account. The API has been enhanced as follows:
    1. If you have user management permissions, you can list all the Personal Access Tokens in your account. You can also filter tokens belonging to a user or filter only active tokens.
    2. If you have service account management permissions, you can list all the service account tokens in your account. You can also filter tokens for a service account or filter only active tokens. (PL-31870, ZD-40110)
+ 
+- When Helm Deployment occurred and the Helm Deploy step timed out during Helm Steady State check, the helm rollback failed to initialize. This is fixed and now the rollback does execute. (CDS-73264). Please make sure the delegate is on 80100 for the changes.
```
Comment (michaelcretzman) [inline release-notes/delegate.md:44]:
```suggestion
- During a Helm deployment, the Helm Deploy step timed when performing the Helm Steady State check and the Helm rollback failed to initialize. This is fixed and now the rollback does execute. (CDS-73264). Please make sure the delegate is on 80100 for the changes.
```

**Review by michaelcretzman — APPROVED** [review]
Approved with one suggestion

File: release-notes/delegate.md (hunk 1 above)
Comment (ershadalim) [reply release-notes/delegate.md:44]: @thisrohangupta This fix is behind feature flag and should be mentioned in documentation as well

Comment (brian-f-harness) [reply release-notes/delegate.md:44]: @thisrohangupta @markoharness  @ershadalim @bharatgoelharness This was documented under the 79906 hotfix on July 20. Do we need to edit the release note with this content? Can we close this PR? Thanks.

**Comment by brian-f-harness** [comment]
Confirmed with @thisrohangupta that this is okay to close.

**Pull request closed without merging**